torch/_torch_docs.py

```diff
@@ -5331,6 +5331,40 @@ def merge_dicts(*dicts):
             [[-0.8437,  0.1727, -0.1398]]])
 """.format(**common_args))
 
+add_docstr(torch.swapdim, r"""
+swapdim(input, dim1, dim2) -> Tensor
+
+Swaps the dimension of :attr:`input` at the position in :attr:`dim1`
```

Comment:
This should just say it's an alias for transpose and link to that documentation.

Reply:
This section of the documentation still needs to be deleted. See, for example, the documentation for subtract: https://pytorch.org/docs/master/generated/torch.subtract.html#torch-subtract.